### DISCUSSION TITLE
Dificulties while writing

### DISCUSSION THREAD

**Discussion by andercard0** [massCodeIO/massCode#227] [Q&A] [answered]
Hi all, 

Not sure if this is only me or a bug so I've decided to label it as Q&A. 

As my title implies, while I'm writing code have a real annoying situtation. For example:. 

I'm in the line 31 from 76 lines in total. if, i decide to add or edit line 31 Masscode throw the focus back at the specific line... as if was pressing page up buttom or something like that...

As it is a bit confuse to understand just made a video showing it real quick.

https://user-images.githubusercontent.com/43047877/184351002-3aa9e524-225c-4077-a205-7123ddeab7ed.mp4

Another thing I've noticed is i can't zoom in ou out my code at all.. tried a bunch of know shortcuts/hotkeys but no deal. I've seen it was implemented but is not working here. 

Thanks!

**Comment by antonreshetov** [comment, marked as answer]
Scrolling up looks like a bug here. I'll take a look at it.

**Reply by antonreshetov** [reply]
Fixed in https://github.com/massCodeIO/massCode/pull/228
